source/dotnet/Services/Infrastructure/BasisData/BatchFileManager.cs

```
@@ -87,14 +107,9 @@ private async Task<List<(Uri Url, string EntryPath)>> GetProcessBasisFileUrlsAsy
             var (directory, extension, entryPath) = fileIdentifierProvider(batchId, gridAreaCode);
             var processDataFile = await TryGetDataLakeFileClientAsync(directory, extension).ConfigureAwait(false);
 
-            if (processDataFile == null)
-            {
-                _logger.LogError("Blob matching '{Directory}*{Extension}' not found", directory, extension);
-            }
-            else
-            {
-                processDataFilesUrls.Add((processDataFile.Uri, entryPath));
-            }
+            if (processDataFile == null) continue;
```

comment on why this is acceptable?

---

I changed my mind. Failing fast and hard instead.
Don't want to find out two weeks later that something had actually been missing inside those zips for a long time.